code/src/pages/GameScreen.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import styled from 'styled-components'
+import "nes.css/css/nes.min.css"
+
+import questions, { generateNextQuestion } from '../reducers/questions'
+
+const Header = styled.header`
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 20px;
+`
+
+const GameScreenMainDiv = styled.div`
+  display: flex;
+  flex-direction: column;
+  padding: 5% 10% 15% 10%;
+  text-align: center;
+  background-size: cover;
+  background-position: center;
+  height: 450px;
+  max-width: 1000px;
+  margin: auto;
+  margin-top: 100px;
+`;
```

Comment:
something general about the styling: your game is not responsive, which made playing it on phone a bit of a Labyrinth itself 🙃  Maybe add some media queries and make it easier for the users ✨